examples/radial_interleafed_example.py

```diff
@@ -0,0 +1,125 @@
+# %% [markdown]
+# # Basic example for reconstruction of radial interleafed sampled data
+# This notebook is meant as a first introduction inot mrpro-based
+# reconstruction of radial interleafed data.
+# %%
+import subprocess
+import tempfile
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+import torch
+
+from mrpro.data import CsmData
+from mrpro.data import IData
+from mrpro.data import SpatialDimension
+from mrpro.data._DcfData import DcfData
+from mrpro.data._KData import KData
+from mrpro.data.traj_calculators._KTrajectoryRadial2D import KTrajectoryRadial2D
+from mrpro.operators import SensitivityOp
+from mrpro.operators._FourierOp import FourierOp
+
+# %%
+
+# flag for conversion and saving nifty files
+# If you wish to save the resulting reconstructed image as a nifty fily, set  NIFTI = "True"
+NIFTI = False
+
+# %% [markdown]
+
+# # Download example data
+# The example data is taken from (https://zenodo.org/records/10669837).
+# It entails data acquired using radial-interleafed sequences written in pulseq.
+# %%
+# Create a temporary directory to save the downloaded files
+data_folder = Path(tempfile.mkdtemp())
+print(f'Data will be saved to: {data_folder}')
+
+# Initialize attempt counter and set maximum number of attempts
+attempt = 1
+max_attempts = 5
+
+# Start the download process
+while attempt <= max_attempts:
+    # Zenodo command for downloading
+    zenodo_cmd = f'zenodo_get 10.5281/zenodo.10664974 -o {str(data_folder)}'
+
+    # Execute the Zenodo command
+    out = subprocess.call(zenodo_cmd, shell=True)
+
+    # Check if the download was successful
+    if out == 0:
+        print('Download successful.')
+        break
+    else:
+        print(f'Download failed! Attempt {attempt} of {max_attempts}')
+        attempt += 1
+
+if attempt > max_attempts:
+    # Abort after reaching the maximum number of attempts
+    raise ConnectionError('Zenodo download failed after 3 attempts!')
+
+# %%
+filepath = data_folder
+seq_filename = '20240130_CEST_interleafed_radial_256px_fov256_8mm_50spokes_100ms.seq'
+h5_filename = 'meas_MID00044_FID00370_20240130_CEST_interleafed_radial_256px_fov256_8mm_50spokes_100m.h5'
+# %% [markdown]
+# # Read in data
+# Data is read in using the downloaded .h5 file and a corresponding
+# trajectory calculator (cartesian in this case). Given the trajectory
+# calculator, mrpro will automatically read, sort and validate the data.
+# %%
+data = KData.from_file(
+    ktrajectory=KTrajectoryRadial2D(),
+    filename=f'{filepath}/{h5_filename}',
+)
+# %% [markdown]
+# # Reconstruction
+# Given the loaded data, the Fourier Operator is initialized next,
+# with which the appropriate Fourier Transform is carried out.
+# Furthermore, density compensation and coil sensitivity maps are created to
+# control for varying coil sensitivities ad densities throughout the scan.
```

Comment:
remove "to control for varying coil sensitivities ad densities throughout the scan."